cosmos-gui/src/app.js

```diff
@@ -35,6 +35,7 @@ var mysqlDriver = require('./mysql_driver.js');
 var OAuth2 = require('./oauth2').OAuth2;
 var logger = require('./logger.js');
 var appUtils = require('./app_utils.js');
+var constants = require('constants');
```

Comment:
Its usage seems perfectly fine and proper as a way to use the constants defined in https://github.com/nodejs/node-v0.x-archive/blob/9cbfd6ef51899dd869a8ec809766cf06d14c9a36/src/node_constants.cc :)